models/model.go

```diff
@@ -0,0 +1,285 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// Package models implements all the structs required by application
+package models
+
+import (
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	// "github.com/zhouzhuojie/conditions"
+
+	"github.com/antonmedv/expr/vm"
+
+	"sync"
+)
+
+// Meta struct
+type Meta struct {
+	TableName                 string                              `json:"TableName"`
+	AttrMap                   map[string]interface{}              `json:"attrMap"`
+	ReturnValues              string                              `json:"ReturnValues"`
+	ConditionExpression       string                              `json:"ConditionExpression"`
+	ExpressionAttributeMap    map[string]interface{}              `json:"ExpressionAttributeMap"`
+	ExpressionAttributeNames  map[string]string                   `json:"ExpressionAttributeNames"`
+	ExpressionAttributeValues map[string]*dynamodb.AttributeValue `json:"ExpressionAttributeValues"`
+	Item                      map[string]*dynamodb.AttributeValue `json:"Item"`
+}
+
+// GetKeyMeta struct
+type GetKeyMeta struct {
+	Key          string                              `json:"key"`
```

Comment:
Should it always start with lowercase or uppercase for json field names? Can we have a consistent way?